docker/pipelines_runner.sh

```diff
@@ -73,13 +85,26 @@ function main {
     exit 1
   fi
 
+  # Build up the extra args is they are provided
+  if [[ ! -z "${subnetwork}" ]]; then
+    echo "Adding --subnetwork ${subnetwork} to optional_args"
+    pt_optional_args="${pt_optional_args} --subnetwork projects/${project}/regions/${region}/subnetworks/${subnetwork}"
+    df_optional_args="${df_optional_args} --subnetwork https://www.googleapis.com/compute/v1/projects/${project}/regions/${region}/subnetworks/${subnetwork}"
```

Comment:
I have never used subnetwork so I am not entirely sure why these strings are different.
Also, I am not sure if the value we expect to receive by --subnetwork flag (and then plug it inside those two strings) is a usual way to provide this flag.

I will leave this to your expertise.

Reply:
The Dataflow param format is different from the GCE format.  My thought was to hide that in the script and just allow users to private the "short name" for the subnetwork.

We could very easily take the GCE version and perpend with the `https://www......` for Dataflow.  In my opinion there isn't much difference from an end-user perspective.

Reply:
Sounds good.